eclim-java.el

```diff
@@ -147,7 +147,7 @@ in eclim when appropriate."
              str)))))
 
 (defun eclim--java-parse-method-signature (signature)
-  (flet ((parser3/parse-arg (arg)
+  (cl-flet ((parser3/parse-arg (arg)
```

Comment:
that's very minor but you should adjust the indentation for the whole function, because it changed